COPYING.md

```diff
@@ -0,0 +1,232 @@
+# License Agreement/Contract(s)
+**Severability/salvatorius: If any word, part, portion, provision , of this is or should become illegal, invalid, null or void, rendering this unenforceable in any jurisdiction, that then shall not affect:** 
+
++ the validity or enforceability in that jurisdiction of any other provision of this Agreement; nor the validity or enforceability in any other types of jurisdictions on that or of any other provision of this Agreement.
+- Any other part or character of the license/contract.
```

Comment:
Valid most common character for bullet is `*`

Reply:
Will amend if that is preferred